<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {background-image: url(src/screen.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="send_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#8d4ba6; color: white</string>
    </property>
    <property name="text">
     <string>отправить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="recievers_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #957dab; color: white</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_photo_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#8d4ba6; color: white</string>
    </property>
    <property name="text">
     <string>загрузить фото</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#8d4ba6; color: white</string>
    </property>
    <property name="text">
     <string>сбросить</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #957dab; color: white</string>
    </property>
   </widget>
   <widget class="QLabel" name="text1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Рассылка</string>
    </property>
   </widget>
   <widget class="QLabel" name="text2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Расписание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_schedule_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#8d4ba6; color: white</string>
    </property>
    <property name="text">
     <string>загрузить расписание</string>
    </property>
   </widget>
   <widget class="QLabel" name="photo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>230</width>
      <height>230</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 40); color: white; border: None</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QToolButton" name="small_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 40); color: white; border: None</string>
    </property>
    <property name="text">
     <string>–</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
